src/mcnp/tests/dagmcnp_unit_tests.cpp

```diff
@@ -76,17 +76,17 @@ TEST_F(DAGMCNP5Test, dagmclcad_test) {
 // Test setup outcomes
 TEST_F(DAGMCNP5Test, dagmcinit_test) {
 // expected values from the lcad file // only the cells
-  const char* expected[] = {"1 9 -12.0 imp:n=1 imp:p=1 ",
-                            "2 9 -12.0 imp:n=1 imp:p=1 ",
-                            "3 9 -12.0 imp:n=1 imp:p=1 ",
-                            "4 9 -12.0 imp:n=1 imp:p=1 ",
-                            "5 3 -2.0 imp:n=1 imp:p=1 ",
-                            "6 3 -2.0 imp:n=1 imp:p=1 ",
-                            "7 3 -2.0 imp:n=1 imp:p=1 ",
-                            "8 3 -2.0 imp:n=1 imp:p=1 ",
-                            "9 1 0.022 imp:n=1 imp:p=1 ",
-                            "12 0  imp:n=0 imp:p=0   $ graveyard",
-                            "13 0  imp:n=1 imp:p=1   $ implicit complement"
+  const char* expected[] = {"1 9 -12.0 imp:n=1.000000 imp:p=1.000000 ",
```

Comment:
This isn't a comment from PullRequest, but I think it's worth updating and is in the spirit of this PR.

With C++11 we can now define these expected values using an initializer list rather than the variable-length char array we're using now like so:

```c++
std::vector<std::string> expected_lcad = {"1 9 -12.0 imp:n=1.000000 imp:p=1.000000 ",                                                                 
                                                                  "2 9 -12.0 imp:n=1.000000 imp:p=1.000000 ",
...
```

Happens in a few places here I think.

Reply:
Good idea. I'll search throughout and update.